cns/requestcontroller/kubecontroller/crdtranslator.go

```diff
@@ -38,6 +39,8 @@ func CRDStatusToNCRequest(crdStatus nnc.NodeNetworkConfigStatus) (cns.CreateNetw
 		ncRequest.NetworkContainerid = nc.ID
 		ncRequest.NetworkContainerType = cns.Docker
 		ncRequest.Version = strconv.FormatInt(nc.Version, 10)
+		log.Printf("Set nc request info with SecondaryIPConfigs %v, NetworkContainerid %s, NetworkContainerType %s, NC Version %s",
```

Comment:
I think it should stay here but removing the part of printing SecondaryIPConfigs and change the word Set to Trying set.  SecondaryIPConfigs dosn't have any value here. SecondaryIPConfigs  start getting value from line #71. However, the other 3 variable already finished getting value so that's why I want to print this log here.